views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/providers/mediaRecorder.js

```diff
@@ -78,6 +78,9 @@ define([
                 // save chunks of the recording
                 mediaRecorder.ondataavailable = function ondataavailable(e) {
                     chunks.push(e.data);
+
+                    var blob = new Blob(chunks, {type: mimeType});
```

Comment:
You're ESLint config/IDE should have complain, no ?
In ES5 code, we enforce the following rule: variable declaration should reflect hoisting. 
So variables should be declared at the top of their context : 
```js
mediaRecorder.ondataavailable = function ondataavailable(e) {
    var blob;
    chunks.push(e.data);
    blob = new Blob(chunks, {type: mimeType});                    
    self.trigger('partialblobavailable', [blob]);
};
```